### PR TITLE
quick start for windows with gpu

### PR DESCRIPTION
**Pull request by ivy-lv11**
Add a quick start for windows with gpu

### CODE REVIEW

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Install BigDL-LLM on Windows for Intel GPU
+
+## MTL & iGPU & Arc
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:3]: What do this line mean?

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
+  ```
+  
+### Install `bigdl-llm`
+
+* With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
+  ```bash
+  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  ```
+
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:51]: Add a note what links to https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#install-bigdl-llm-from-wheel if there are network issues when installing IPEX

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+* With the `llm` environment active, use `pip` to install the **OneAPI Base Toolkit**:
+  ```bash
+  pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
+  ```
+  
+### Install `bigdl-llm`
+
+* With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
+  ```bash
+  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:49]: Shall we use `--extra-index-url` (see https://github.com/analytics-zoo/nano/issues/919)? @Oscilloscope98 

Need to verify it on both Intel and non-Intel network

Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:49]: Yes, we can use `--extra-index-url` as the `-f https://developer.intel.com/ipex-whl-stable-xpu` may be deprecated later. We have had some verification before for this `--extra-index-url` on different network. But we could verify them again :)

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 1 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:3]: Change it to a text line "This guide applies to Intel [Meteor Lake](https://en.wikipedia.org/wiki/Meteor_Lake) iGPU and Core 11 and 12 generation iGPU, as well as Intel Arc Series GPU". Is it okay?

Comment (jason-dai) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:3]: Core Ultra
Core 12 - 14 gen

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 2 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:52]: added note

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 1 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md]: revised

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 3 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:49]: @ivy-lv11 please kindly verify this and update the command.

**Review by shane-huang — APPROVED** [review]
LGTM

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ This guide applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs, as
 
 * With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
   ```bash
-  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:49]: Need to add both links (us & cn)

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+   prompt = " Question:{prompt}\n\n Answer:".format(prompt=question)
+   # Generate predicted tokens
+   with torch.inference_mode():
+       input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu')
+       output = model.generate(input_ids, do_sample=False, max_new_tokens=32, generation_config = generation_config).cpu()
+       output_str = tokenizer.decode(output[0], skip_special_tokens=True)
+       print(output_str)
+   ```
+
+* An example output on the laptop equipped with i7 11th Gen Intel Core CPU and Iris Xe Graphics iGPU looks like below.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:90]: How to run this? And any ENV needed (see https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration)

Comment (ivy-lv11) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:90]: Copy/paste the python code and run it in the environment created above. When using `pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0` to install OneAPI, we do not need to add runtime configurations.

Comment (jason-dai) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:90]: 1) Add instructions
2) I think `SYCL_CACHE_PERSISTENT` and `BIGDL_LLM_XMX_DISABLED` need to be configured differently for different GPU @Oscilloscope98 ?

Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:90]: Yes, we have different configuration for different GPU. Detailed are in here: https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 4 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:49]: @ivy-lv11 please help verify if `pip install --pre --upgrade bigdl-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/` works.

File: docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md (hunk 5 above)
Comment (shane-huang) [reply docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md:90]: submitted a new PR to address these revisions:
- quick example instructions step-by-step
- add run time configuration step
- add us/cn extra index url specs

**Pull request merged**